lib/tools.js

```diff
@@ -73,7 +74,7 @@ function asString (str) {
   if (l > 100) {
     return JSON.stringify(str)
   }
-  for (var i = 0; i < l && point >= 32; i++) {
```

Comment:
Has something changed with `var` vs `let` in loops?

Reply:
I just ran standard --fix on this file. I'll revert, good spot.

Reply:
Use `eslint --fix`. We are using that here so we don't have to liter the whole codebase with pragma directives specifically because of `var/let`.